$DragonFly: src/lib/libarchive/patches/Attic/archive_read_support_format_all.c.patch,v 1.1 2007/04/11 22:49:22 corecode Exp $
Index: archive_read_support_format_all.c
===================================================================
RCS file: /home/dcvs/src/contrib/libarchive-2.0/libarchive/archive_read_support_format_all.c,v
retrieving revision 1.1.1.2
diff -u -p -r1.1.1.2 archive_read_support_format_all.c
--- archive_read_support_format_all.c	7 Apr 2007 14:04:55 -0000	1.1.1.2
+++ archive_read_support_format_all.c	11 Apr 2007 22:34:40 -0000
@@ -31,7 +31,6 @@ 
 int
 archive_read_support_format_all(struct archive *a)
 {
-	archive_read_support_format_ar(a);
 	archive_read_support_format_cpio(a);
 	archive_read_support_format_empty(a);
 	archive_read_support_format_iso9660(a);
